docs/components/databases.rst

```diff
@@ -62,18 +75,32 @@ How to use the :code:`mysql` or :code:`mariadb` format?
     PYFUNCEBLE_DB_USERNAME=pyfunceble
 
 .. note::
-    Since the version :code:`2.4.3.dev` it have possible to provide a UNIX socket
-    file for the :code:`PYFUNCEBLE_DB_HOST` environment variable.
+    Since version :code:`2.4.3.dev` it is possible to use the UNIX socket
+    for the :code:`PYFUNCEBLE_DB_HOST` environment variable.
 
-    The typycal location for :code:`mysqld.sock` is :code:`/var/run/mysqld/mysqld.sock`.
+    The typical location for :code:`mysqld.sock` is 
+    :code:`/var/run/mysqld/mysqld.sock`.
     This have been done to make:
 
-      1. It easier to use in conjunction with a supported CI environment/platform.
-      2. Leaving more space on the IP-stack on local DB installations
-      3. The :code:`UNIX:SOCKET` is usually faster than the IP connection on local runs
+      1. It easier to use it in conjunction with a supported CI 
+	  environment/platform.
+      2. Leaving more space on the IP-stack on local DB installations.
+      3. The :code:`UNIX:SOCKET` is usually faster than the IP connection on 
+	  local runs.
 
-4. Switch the :code:`db_type` index of your configuration file to :code:`mysql` or :code:`mariadb`.
+::
```

Comment:
Please intend the code block as it is part of the note.

Reply:
The codeblock is now indented.  Nicely spotted, thanks

https://github.com/funilrys/PyFunceble/pull/72/files#diff-028ae80d510abbabb76f10b7060c48a1R91-R98